### ISSUE TITLE
format string errors in .po files

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#698]
```
msgfmt -c}}} complains about several format string errors in the .po files. The attached patch should fix all of them.
{{{msgfmt -c
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/698
- Originally reported by: Jakub Wilk
- Originally created at: 2011-05-18T17:13:43.356

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 07:47:55+00:00_

Fix #698: msgstr errors found by msgfmt -c.

→ <<cset b9bd2ff791ce5195668a4ffa8f552bb3b3cb402d>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 08:47:51+00:00_

Fix #698: msgstr errors found by msgfmt -c.

→ <<cset 08d556d4b6f051fb75f73ce4f77d439771abdb19>>
